core/aws_ddk_core/stages/glue_transform.py

```diff
@@ -155,8 +167,15 @@ def __init__(
             },
             iam_resources=[crawler_arn],
         )
+
         success = Succeed(self, "success")
-        crawl_object.add_catch(success, errors=["Glue.CrawlerRunningException"])
+
+        crawl_object.add_retry(
```

Comment:
So in high-concurrency environments would we expect this to fail relatively often? Also, do you think there's value in adding some override to trigger crawler but not worry about failures due to already running errors?

Reply:
Added an explicit parameter to drive behavior